daffodil-cli/src/main/scala/org/apache/daffodil/Main.scala

```diff
@@ -1400,8 +1408,15 @@ object Main {
 
   private def unparseWithSAX(
     is: InputStream,
-    contentHandler: DFDL.DaffodilUnparseContentHandler): UnparseResult = {
-    val xmlReader = DaffodilSAXParserFactory().newSAXParser.getXMLReader
+    contentHandler: DFDL.DaffodilUnparseContentHandler,
+    infosetType: InfosetType.Type): UnparseResult = {
+    val xmlReader = infosetType match {
+      case InfosetType.EXI => {
+        val exiSource = new EXISource()
+        exiSource.getXMLReader
+      }
+      case _ => DaffodilSAXParserFactory().newSAXParser.getXMLReader
```

Comment:
What was the reason for removing SAX support aside from it's a little unintuitive since it's an API, very similar to JDOM, Scala-XML, W3CDOM, etc, that we do still support? It looks like the way you've made the changes SAX should all still work. In fact, right here we still have support for normal SAX.